saw-remote-api/python/tests/saw/test_basic_java.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+from pathlib import Path
+
+import saw_client as saw
+
+from saw_client.jvm import Contract, java_int, cryptol
+
+class Add(Contract):
+    def __init__(self) -> None:
+        super().__init__()
+
+    def specification(self) -> None:
+        x = self.fresh_var(java_int, "x")
+        y = self.fresh_var(java_int, "y")
+
+        self.execute_func(x, y)
+
+        self.returns(cryptol("(+)")(x,y))
+
+class AddTest(unittest.TestCase):
```

Comment:
This test looks like it uses 2 of the 3 added server methods. Is it worth adding another test or modifying this one so it also does a quick sanity check for `SAW/JVM/assume`?

Reply:
Done.